modules/dynamic-plugins/proc-using-topology-plugin.adoc

```diff
@@ -2,19 +2,21 @@
 Topology is a front-end plugin that enables you to view the workloads as nodes that power any service on the Kubernetes cluster.
 
 .Prerequisites
-* You have installed the Red Hat Developer Hub (RHDH).
+* Your {product} instance is installed and running.
 * You have installed the Topology plugin. 
 //For the installation process, see Installation.
-* If the RBAC permission framework is enabled, ensure to add the following permission policies in an external permission policies configuration file named `rbac-policy.csv` to allow the RBAC admins or your desired user(s)/group(s) to access the Topology plugin:
+* If the RBAC permission framework is enabled, ensure the following permission policies are added in an external permission policies configuration file, such as `rbac-policy.csv`, to allow the RBAC admins or authorized users to access the Topology plugin:
 +
 [source,bash]
```

Comment:
This is a CSV file, not a bash script. The source highlighting is not effective, and it is confusing for the source code reader. Consider removing it.

```suggestion
[source]
```